libseccomp/src/filter_context.rs

```diff
@@ -541,6 +632,21 @@ impl ScmpFilterContext {
     ///
     /// If this function is called with an invalid filter, an issue is encountered
     /// getting the current state, or the libseccomp API level is less than 4, an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// let mut ctx = ScmpFilterContext::new_filter(ScmpAction::Allow)?;
+    /// if check_api(4, ScmpVersion::from((2, 5, 0))).unwrap() {
```

Comment:
Could you hide these branches using `#` ?
I think it will be good example code if you can hide the branch.
The same is true of other example code.